src/vs/workbench/contrib/externalUriOpener/common/configuration.ts

```diff
@@ -64,6 +64,19 @@ export const externalUriOpenersConfigurationNode: IConfigurationNode = {
 	}
 };
 
+export const defaultBrowserNode: IConfigurationNode = {
+	...workbenchConfigurationNodeBase,
+	properties: {
```

Comment:
I wonder about the name, this is not really the default browser, but the actual browser that is used? We could call it `workbench.browser` or `application.browser` 🤔